### PR TITLE
SC-3163 : Added a popup code to remove a sensitive key from the prope…

### PR DESCRIPTION
**Pull request by vivekdf**
Made changes to remove sensitive information getting appended to the log file, during properties write operation.

### CODE REVIEW

File: projects/vdk-core/src/vdk/internal/builtin_plugins/job_properties/datajobs_service_properties.py [hunk 1]
```diff
@@ -78,6 +78,8 @@ def set_all_properties(self, properties: Dict[str, PropertyValue]) -> None:
             )  # keeps the outer scope originally-referenced dictionary intact
             for client in self._write_preprocessors:
                 try:
+                    if (properties.get("cipher_key")) is not None:
```
Comment (DeltaMichael) [inline projects/vdk-core/src/vdk/internal/builtin_plugins/job_properties/datajobs_service_properties.py:81]: This prevents the key from getting passed to the backend service. It won't get persisted at all. I don't think this is what we want here.

**Review by DeltaMichael — CHANGES_REQUESTED** [review]
Please provide more information about the exact problem you're trying to solve. Your commit message and PR description should follow this template. https://github.com/vmware/versatile-data-kit/blob/main/support/git-commit-template.txt

Please check our contributing guide for more information. https://github.com/vmware/versatile-data-kit/blob/main/CONTRIBUTING.md#submit-a-pull-request

File: projects/vdk-core/src/vdk/internal/builtin_plugins/job_properties/datajobs_service_properties.py (hunk 1 above)
Comment (dakodakov) [reply projects/vdk-core/src/vdk/internal/builtin_plugins/job_properties/datajobs_service_properties.py:81]: Agreed.

**Comment by DeltaMichael** [comment]
Closing due to inactivity.

**Pull request closed without merging**